public/index.html

```diff
@@ -23,12 +23,8 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <link href="bootstrap/css/bootstrap.css" rel="stylesheet">
     <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"
```

Comment:
including the integrity bit is a good idea for security.

Reply:
@mjkkirschner I assume it will be same as package-lock.json? Will update

Reply:
@mjkkirschner I cant find the integrity attribute, I found using the one in package-lock.json did not work

Reply:
You can generate it via https://www.srihash.org/